webauthn.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "cbor", "~> 0.5.9.2"
   spec.add_dependency "cose", "~> 0.1.0"
+  spec.add_dependency "jwt", "~> 2.0"
```

Comment:
I'm bit aware of using `~>` in a library. This could make a conflict on a RP implementation with other gems. These constraints should be used only when we acknowledged the incompatibility.

Reply:
Good call, https://rubygems.org/gems/jwt/reverse_dependencies lists `oauth2` as a popular dependency tied to `~> 1.0`.